src/Update/Updater.php

```diff
@@ -479,6 +479,30 @@ public function moveFile($source, $target, $overwrite = FALSE) {
   }
 
   /**
+   * Copies a file from the BLT template to the repository.
+   *
+   * @param string $source
+   *   The filepath, relative to the BLT template directory.
+   *
+   * @return bool
+   *   FALSE if nothing happened.
+   */
+  public function syncWithTemplate($filePath, $overwrite = FALSE) {
+    $sourcePath = $this->getBltRoot() . '/template/' . $filePath;
+    $targetPath = $this->getRepoRoot() . '/' . $filePath;
+    $fileCopied = FALSE;
+
+    if ($this->getFileSystem()->exists($sourcePath)) {
+      $this->getFileSystem()->copy($sourcePath, $targetPath, $overwrite);
+      if ($overwrite) {
```

Comment:
@grasmash that's true. Good catch. Looking back at the [fileSystem copy method](http://symfony.com/doc/current/components/filesystem.html#copy), I realized we actually don't have a way of knowing if the file actually copied if `$overwrite` is `FALSE` without us manually comparing the last modified date on the target file and the source file. I can either modify the logic to check that information (which feels a bit excessive) or I can simply not return anything - which is what the `fileSystem` copy method is currently doing (doesn't feel great either). Thoughts?